### PR TITLE
Add initial translation for auth-x509-client-username-form 

### LINKED ISSUE #29868: Missing Text for x509
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

translations

### Describe the bug

Translation for placeholder "auth-x509-client-username-form-display-name" is missing for English and German and possibly any other language 

### Version

24.0.4

### Regression

- [ ] The issue is a regression

### Expected behavior

A Display Name like "X509 Certificate" or in GER "X509 Authentifizierung"

### Actual behavior

It shows in "Try another way" "auth-x509-client-username-form-display-name" with Help Text "auth-x509-client-username-form-help-text"

### How to Reproduce?

Add "X509/Validate Username Form" to Browser Form as Alternative. Start login to realm. Select "Try another way". 

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by thomasdarimont**
Fixes #29868

### CODE REVIEW

**Review by edewit — DISMISSED** [review]

File: themes/src/main/resources/theme/base/login/messages/messages_en.properties [hunk 1]
```diff
@@ -433,6 +433,8 @@ auth-username-form-display-name=Username
 auth-username-form-help-text=Start sign in by entering your username
 auth-username-password-form-display-name=Username and password
 auth-username-password-form-help-text=Sign in by entering your username and password.
+auth-x509-client-username-form-display-name=X509 Certificate
+auth-x509-client-username-form-help-text=Sign in with a X509 client certificate
```
Comment (ahus1) [inline themes/src/main/resources/theme/base/login/messages/messages_en.properties:437]: Should "an" as it is an "Ex-Five-Oh-Nine" certificate, and that starts with a vowel. 

```suggestion
auth-x509-client-username-form-help-text=Sign in with an X509 client certificate
```

**Review by ahus1 — CHANGES_REQUESTED** [review]
Thank you for adding those missing keys. See below for a small grammar issue that seems to lurk in there.

**Review by ahus1 — DISMISSED** [review]

**Review by jonkoops — DISMISSED** [review]

**Review by ahus1 — CHANGES_REQUESTED** [review]
This PR now contains a change to `requiredAction.organization.confirm-membership` which seems to be unrelated. Please comment if it is really related, otherwise remove that change. 

Unfortunately due to the latest changes to `.editorconfig` there are some blank removed that we want to keep for now. See #29982 for a discussion.

**Comment by thomasdarimont** [comment]
IntelliJ seems todo some strange automatic cleanup for spaces at the end? I'm just saving the file.

**Comment by thomasdarimont** [comment]
Fixed

**Review by agagancarczyk — APPROVED** [review]
LGTM!

**Review by ahus1 — APPROVED** [review]

**Pull request merged**